exercises/kindergarten-garden/uKindergartenGarden.pas

```diff
@@ -0,0 +1,61 @@
+unit uKindergartenGarden;
+
+interface
+
+uses
+  System.Generics.Collections;
+
+type
+  TGarden = class
+  private
+    FArr : TArray<string>;
+    FStudents : TList<string>;
+    FPlants : TDictionary<char, string>;
+
```

Comment:
@filiptoskovic this looks really good.  I am going to mark some things that I would like you to fix.  So far everything is very superficial white space stuff.

Delete line 14.